src/store/actions.ts

```diff
@@ -58,30 +58,36 @@ export default {
   }),
   loadGridVariables: tryAction(async ({ state, commit } : any) => {
     commit('updateGridVariables', [])
-    const response = await api.get(`/api/v2/lifelines_subsection_variable?q=subsection_id==${state.treeSelected}&attrs=~id,id,subsection_id,variable_id(id,name,label,variants(id,assessment_id))&num=10000`)
-    commit('updateGridVariables', response.items
+    const subsectionId = state.treeSelected
+    const response = await api.get(`/api/v2/lifelines_subsection_variable?q=subsection_id==${subsectionId}&attrs=~id,id,subsection_id,variable_id(id,name,label,variants(id,assessment_id))&num=10000`)
+    if (state.treeSelected === subsectionId) {
+      commit('updateGridVariables', response.items
       // map assessment_id to assessmentId somewhere deep in the structure
-      .map((sv: any) => ({
-        ...sv.variable_id,
-        variants: sv.variable_id.variants
-          .map((variant: any) => ({
-            ...variant,
-            assessmentId: variant.assessment_id
-          }))
-      })))
+        .map((sv: any) => ({
```

Comment:
what does ‘sv’ stand for ?

Reply:
code is unchanged, just gets tabbed one more indentation but stands for subsection variable